### ISSUE TITLE
bf luis:test command not working on Mac OS X

### ISSUE THREAD

**Issue by GrazFuchs** [microsoft/botframework-cli#1320]
## Versions
CLI Version 4.14.1
Nodejs version 14.15.0
Command-line interpreters: Bash
Mac OS X Bis Sur 11.6

## Describe the bug
When I try to test my LUIS App I'll get the following error message:

```
Unknown error during execution. Please file an issue on https://github.com/microsoft/botframework-cli/issues
Access denied due to invalid subscription key or wrong API endpoint. Make sure to provide a valid key for an active subscription and use a correct regional API endpoint for your resource.
```
Command was 
`bf luis:test -s "<subscription key>" -a "<app id>" -i "<testfile>.lu"`

## Expected behavior
I should get a testresponse from LUIS service.

[bug]

**Comment by johnataylor** [comment]
@anishprasad01 could you take a look, note this has been reported on Mac OS

**Comment by anishprasad01** [comment]
Hi @GrazFuchs

I attempted reproduce your issue on my machine and I was able to successfully receive a test response from the LUIS application. 

However, I noticed one potential easy mistake that might be causing your issue. When you filled in your subscription key for the `-s/subscriptionKey` parameter, did you use your **Prediction** resource key, or your **Authoring** resource key? When the LUIS app specifies "subscription" key, it is referring to the Authoring resource, which represents your "subscription" to author and publish LUIS apps. If you supply the `s/subscriptionKey` parameter with your Prediction key, you'll get the error message you posted.

Try running the test again while making doubly sure you're using the `Authoring` key. It also wouldn't hurt to make sure your `appId` is correct, and that your LUIS app has been published in the `Production` slot before attempting to reach it.

For reference, your Authoring keys can be found in one of two places:
- On the luis.ai portal under `<Your App> -> Manage -> Azure Resources -> Authoring Resource -> Primary Key`
- In the Azure Portal in the `Keys and Endpoint` blade of your Authoring resource
